programs/compressed-token/src/process_transfer.rs

```diff
@@ -557,6 +584,8 @@ pub mod transfer_sdk {
             pubkey_array,
```

Comment:
pubkey array has been deleted from this struct onchain but still exists here.
Happy to add the accounts to remaining accounts shouldn't be a big change.